src/pkcs1v15.rs

```diff
@@ -20,60 +19,52 @@ use crate::errors::{Error, Result};
 use crate::key::{self, PrivateKey, PublicKey};
 use crate::{RsaPrivateKey, RsaPublicKey};
 
-#[derive(Clone)]
+#[derive(Clone, PartialEq, Eq)]
 pub struct Signature {
-    bytes: Vec<u8>,
+    bytes: Box<[u8]>,
```

Comment:
Though this is ok as a first pass, I think ideally you could replace the internal representation with a `BigUint` now, which would eliminate the need for an intermediate `Box<[u8]>` when signing/verifying.

Reply:
Ack, I'll take a look on it later.

Reply:
Opened #220 . Implementing it would require changing the internals of the whole RSA crate.

Reply:
Thanks